docs/100-upgrade/migration-guides/3.2-3.3.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: 3.2 -> 3.3
+description:
+  This page lists the highlights for upgrading a project from Front-Commerce 3.2
+  to 3.3.
+---
+
+<p>{frontMatter.description}</p>
+
+## Update dependencies
+
+Update all your `@front-commerce/*` dependencies to this version:
+
+```shell
+pnpm update "@front-commerce/\*@3.3.0"
+```
+
+## Code changes
+
+### Remix entrypoints
+
+In this release, we added new features that required to hook into your Remix
+application entrypoints created with the initial skeleton.<br /> You must update
```

Comment:
**nitpick:** it's not necessary new features.

**suggestion:** use a broader sentence format

```suggestion
In this release, we improved features that required to hook into your Remix
application files created with the initial skeleton.<br /> You must update
```